javascript/embedded-login/src/main.js

```diff
@@ -10,17 +10,25 @@ import * as forgerock from '@forgerock/javascript-sdk';
  * of the MIT license. See the LICENSE file for details.
  */
 
+const wellknownUrl = process.env.SERVER_URL.endsWith('/')
+  ? `${process.env.SERVER_URL}oauth2/${process.env.REALM_PATH}/.well-known/openid-configuration`
+  : `${process.env.SERVER_URL}/oauth2/${process.env.REALM_PATH}/.well-known/openid-configuration`;
+
 const FATAL = 'Fatal';
-forgerock.Config.set({
-  clientId: process.env.WEB_OAUTH_CLIENT, // e.g. 'ForgeRockSDKClient'
-  redirectUri: `${window.location.origin}/callback.html`, // e.g. 'https://sdkapp.example.com:8443/callback.html'
-  scope: process.env.SCOPE, // e.g. 'openid profile email address phone'
+
+/**
+ * CUSTOMERS:
+ * Replace the argument passed into `setAsync` below with your own sample config object from
+ * your PingAIC/PingAM's OAuth configuration.
+ */
+await Config.setAsync({
+  clientId: process.env.WEB_OAUTH_CLIENT, // e.g. 'PingSDKClient'
+  redirectUri: `${window.location.origin}/callback.html`, // e.g. 'https://localhost:8443/callback.html'
+  scope: process.env.SCOPE, // e.g. 'openid profile email'
   serverConfig: {
-    baseUrl: process.env.SERVER_URL, // e.g. 'https://myorg.forgeblocks.com/am' or 'https://openam.example.com:8443/openam'
-    timeout: 3000, // 90000 or less
+    wellknown: wellknownUrl, // This can be found in your PingAM's OAuth2 configuration
   },
   realmPath: process.env.REALM_PATH, // e.g. 'alpha' or 'root'
```

Comment:
I don't believe we need realm path if we are using the wellknown url. However, we should probably include a `tree` param for the journey name.

Reply:
Yes, you are right. I'll remove it now.